### PR TITLE
Sc 3853/looker download dashboard as file

### PR DESCRIPTION
**Pull request by Shamwayne**

### CODE REVIEW

**Comment by smjohnsonShipyard** [comment]
Getting this error when trying to run this blueprint. Trying to pass in 11 as the dashboard_id
image

**Comment by smjohnsonShipyard** [comment]
I'm getting an error on the import of sdk_exceptions. Are you getting this on your side @Shamwayne? 
image

**Comment by Shamwayne** [comment]
Fixed, had to do with the documentation/tutorial i was using for looker being from an outdated version

**Review by smjohnsonShipyard — APPROVED** [review]

File: looker_blueprints/download_dashboard.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            )
+            sys.exit(EXIT_CODE_LOOK_DASHBOARD_ERROR)
+        elif poll.status == "success":
+            break
+
+        time.sleep(delay)
+        elapsed += delay
+    print(f"Render task completed in {elapsed} seconds")
+
+    result = sdk.render_task_results(task.id)
```
Comment (ericelsken) [inline looker_blueprints/download_dashboard.py:75]: On line 61, is that the right sdk call? It seems more like a verb `render_task` than a getter. 

How fast is Looker expected to complete the render. Delaying for half a second per request seems like way too little time to wait. I would normally expect polling to be on the 5 to 10 second scale. Is there documentation stating you can poll this fast?  [](edited-with-Axolo)

**Review by ericelsken — CHANGES_REQUESTED** [review]

File: looker_blueprints/download_dashboard.py (hunk 1 above)
Comment (Shamwayne) [reply looker_blueprints/download_dashboard.py:75]: Using the example provided by looker itself here:

https://github.com/looker-open-source/sdk-codegen/blob/main/examples/python/download_dashboard_pdf.py

**Pull request merged**